lib/entities/state/attributes/CarpetTurboStateAttribute.js

```diff
@@ -0,0 +1,32 @@
+const StateAttribute = require("./StateAttribute");
```

Comment:
Since this is just a setting, it should not be a state attribute but rather a simple entity in `core/entities`

Reply:
This is just a hack to make it stick around since Viomi does not have a command to retrieve the setting, by adding it to the status it can at least be remembered as long as Valetudo is running.

I know it's not ideal, I'm not sure if there's a better way to do it. I'm also not sure what you have in mind with the entities.

I can remove it completely though and always return an error, if you think it's better.